Tests/APITesters/SwiftAPITester/SwiftAPITester/VerificationResultAPI.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 import RevenueCat
 
-func checkVerificationResultAPI(_ mode: EntitlementVerificationMode = .disabled,
+func checkVerificationResultAPI(_ mode: Configuration.EntitlementVerificationMode = .disabled,
```

Comment:
This is technically the correct way. I don't understand why it compiles without it. CMD+clicking it finds the Obj-C symbol 🤔